draft-ietf-homenet-front-end-naming-delegation.md

```diff
@@ -392,13 +393,14 @@ As some deployments may not provide a DOI that will be able to update the DS in
 By accepting the DS RR, the DM commits to advertise the DS to the parent zone.
 On the other hand if the DM does not have the capacity to advertise the DS to the parent  zone, it indicates this by refusing the update to the DS RR.
 
-## Information to set the Synchronization Channel {#sec-sync-info}
+## Information to set up the Synchronization Channel {#sec-sync-info}
 
 The HNA works as a hidden primary authoritative DNS server, while the DM works like a secondary.
 As a result, the HNA must provide the IP address the DM should use to reach the HNA.
-If the HNA detects that it has been renumbered, then it MUST use the Control Channel to update the DOI with its new IPv4 and IPv6 addresses.
 
-The Synchronization Channel will be set between that IP address and the IP address of the DM.
+If the HNA detects that it has been renumbered, then it MUST use the Control Channel to update the DOI with the new IPv6 address it has been assigned.
+
+The Synchronization Channel will be set between the new IPv6 address and the IP address of the DM.
```

Comment:
I suspect we should probably only say the new IP instead of the new IPv6

Reply:
Do we support Synchronization over IPv4?

Reply:
The DNS mechanism does not mandate IPv6, so I have been asked to not prevent IPv4 and as far as I can see we do support IPv4.